lib/screens/config/v2/pre_fare.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Screens.Config.V2.PreFare do
+  @moduledoc false
+
+  @type t :: %__MODULE__{
+          station_id: Screens.Stops.Stop.id()
```

Comment:
Following the convention from [bus shelter](https://github.com/mbta/screens/blob/master/lib/screens/config/v2/bus_shelter.ex) config, this struct can just be empty for now! We will add keys to it corresponding to each widget that can appear on pre-fare screens. (So probably, `header: CurrentStopId.t()` and `elevator_status: ElevatorStatus.t()` pretty soon)